app/src/main/java/com/swent/assos/model/view/LoginViewModel.kt

```diff
@@ -34,11 +35,13 @@ constructor(private val storageService: DbService, private val accountService: A
               user =
                   User(
                       document.id as String,
-                      document.data?.get("firstname") as String,
-                      document.data?.get("name") as String,
-                      document.data?.get("email") as String,
-                      document.data?.get("associations") as List<Triple<String, String, Int>>,
-                      document.data?.get("following") as List<String>)
+                      (document.data?.get("firstname") ?: "") as String,
+                      (document.data?.get("name") ?: "") as String,
+                      (document.data?.get("email") ?: "") as String,
+                      (document.data?.get("associations")
+                          ?: emptyList<Triple<String, String, Int>>())
+                          as List<Triple<String, String, Int>>,
```

Comment:
check the firebase collection users, the current type of associations is a list of string